couchbase-on-ubuntu/azuredeploy.parameters.json

```diff
@@ -12,19 +12,19 @@
       "value": "Small"
     },
```

Comment:
@rafaelgodinho See how to enable placeholder parameters to allow the CI to run this template https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci